pkg/proto/aksnodeconfig/v1/tlsbootstrappingconfig.pb.go

```diff
@@ -20,17 +20,78 @@ const (
 	_ = protoimpl.EnforceVersion(protoimpl.MaxVersion - 20)
 )
 
+type BootstrappingAuthMethod int32
+
+const (
+	//nolint:gosec // this is a const string to use in switch statements, not hardcoded credentials
+	BootstrappingAuthMethod_UseTLSBootstrapToken      BootstrappingAuthMethod = 0
+	BootstrappingAuthMethod_UseSecureTLSBootstrapping BootstrappingAuthMethod = 1
+	BootstrappingAuthMethod_UseArcMsiToMakeCSR        BootstrappingAuthMethod = 2
+	BootstrappingAuthMethod_UseAzureMsiToMakeCSR      BootstrappingAuthMethod = 3
+	BootstrappingAuthMethod_UseArcMsiDirectly         BootstrappingAuthMethod = 4
+	BootstrappingAuthMethod_UseAzureMsiDirectly       BootstrappingAuthMethod = 5
+)
+
+// Enum value maps for BootstrappingAuthMethod.
+var (
+	BootstrappingAuthMethod_name = map[int32]string{
+		0: "UseTLSBootstrapToken",
+		1: "UseSecureTLSBootstrapping",
+		2: "UseArcMsiToMakeCSR",
+		3: "UseAzureMsiToMakeCSR",
+		4: "UseArcMsiDirectly",
+		5: "UseAzureMsiDirectly",
+	}
+	BootstrappingAuthMethod_value = map[string]int32{
```

Comment:
what is this used for ?

Reply:
protobuf unmarshalling. This is autogenerated toce.